tests/unit_tests/data_steward/utils/pipeline_logging_test.py

```diff
@@ -32,16 +31,26 @@ def setUpClass(cls):
     def setUp(self):
         self.log_file_list = ['path/', 'faked.log', 'path/fake.log']
         self.log_path = [
-            datetime.now().strftime('path/curation%Y%m%d_%H%M%S.log'),
-            'logs/faked.log', 'path/fake.log'
+            'path/curation20201023_010101.log', 'logs/faked.log',
+            'path/fake.log'
         ]
         generate_paths(self.log_file_list)
 
+    @mock.patch('utils.pipeline_logging.datetime')
     @mock.patch('utils.pipeline_logging.generate_paths')
-    def test_generate_paths(self, mock_generate_paths):
+    def test_generate_paths(self, mock_generate_paths, mock_datetime):
+        # mocking datetime to make sure slight increase in time won't cause a failure
+        mock_datetime.now = mock.Mock(
+            return_value=datetime(2020, 10, 23, 1, 1, 1, 1))
+
         # checks that log_path is generated properly
         results = generate_paths(self.log_file_list)
-        self.assertEquals(results, self.log_path)
+
+        # tests if the lists are equal regardless of order
+        self.assertCountEqual(results, self.log_path)
+
+        # tests if the lists are equal
+        self.assertEqual(results, self.log_path)
```

Comment:
nit: This is possibly redundant since there is a check for `self.assertListEqual` in the next line

Reply:
I can get rid of `self.assertEqual(results, self.log_path)` 👍🏾